### DISCUSSION TITLE
.com泡沫: 一次科技的金融泡沫事件

### DISCUSSION THREAD

**Discussion by Tomlord1122** [1101-GS4719-C/Python-Community-GS4719-C#190] [科技新知]
.com泡沫是在1995年至2001年間與資訊科技及網際網路相關的投機泡沫事件，網際網路及資訊科技相關企業的股價高速上升，在2000年3月10日那斯達克指數觸及5,408.60[2]的最高點時達到頂峰，且以5,048.62收盤。在此期間，西方國家的投機者看到網際網路板塊及相關領域的快速增長，最具代表性的就是很多以「.com」的網際網路公司不斷成立。公司可以簡單地通過在他們名字上添加「e-」字首或是「.com」的字尾來使其股票價格增長。
今天會想要分享這個，是因為最近在幣圈有不少人認為這是再一次重現.com泡沫的可能，虛擬貨幣的一直漲，大家不免懷疑不斷的成長，這個虛擬的東西現階段對社會到底有甚麼好處，因此開始有人預估不久後，虛擬貨幣會迎來一次非常大的起伏，泡沫化等等，但我認為這是一定會發生的，只是不知道是什麼時候，有可能是幾個月，或是一兩年吧，雖然是這樣認為，但我還是非常看好區塊鏈這項技術，幾年後我想不僅是金融體系，甚至一般民眾的生活模式都會有巨大的變革

**Comment by Jie2222** [comment]
虛擬貨幣的價值建立在區塊鏈的技術上，雖然區塊鏈的技術大幅增加了可靠性，但同時也因此讓一但發生問題處理起來會特別複雜，會不會泡沫化真的很難說

**Reply by chia-kai-chang** [reply]
山寨幣可能會泡沫化，但是大幣(BTC, ETH)應該是不會~ 我猜~

**Comment by HSU-YU-JEN** [comment]
引用查理蒙格對於加密貨幣的看法
"我永遠不會購買加密貨幣，我希望他們從未被發明出來。我認為中國人做出了正確的決定，那就是簡單有力的禁止加密貨幣，我的國家（美國）做了錯誤決定。"
